### ISSUE TITLE
Kohana 3.2 support missing.

### ISSUE THREAD

**Issue by miket3** [mixu/useradmin#15]
This system as it stands now does not work 100% in Kohana 3.2.     ANY and ALL functions that receive parameters must be upgraded to include         $p1 = $this->request->param(); otherwise any passed parameters will be ignored.  There may be other upgrades necessary but I am just diving in now.  If I ever figure out how to use git  I will certainly fork and commit my upgrades. Just one example is:
Useradmin/Controller/Admin/user.php->Action_Edit($id = NULL)

You must add the following line:
$id   = $this->request->param('id','');

Otherwise the form will not be able to show the user data.

**Comment by chrix98** [comment]
yes, there's a number of changes to be made to port useradmin to 3.2, one of which you've mentioned (see http://kohanaframework.org/3.2/guide/kohana/upgrading#controller-action-parameters) ...

There's some pretty good comments by Enigma (on http://blog.mixu.net/2011/01/13/getting-started-with-useradmin-my-kohana-3-auth-admin-module/ ) who's listed most of the changes required to port useradmin to 3.2 - his list might not be complete though

**Comment by eblount** [comment]
If it helps, I've made these changes in my fork on GitHub: https://github.com/eblount/useradmin
